theano/gpuarray/extra_ops.py

```diff
@@ -22,8 +22,9 @@ class GpuCumsum(GpuKernelBase, Op):
     SUPPORTED_NDIMS = 3
     __props__ = ('axis',)
 
-    def __init__(self, axis):
```

Comment:
oh! This should have gotten removed from the rebase. It existed before, but i have changed this behaviour in #4570 itself